docs/experiments.md

```diff
@@ -0,0 +1,30 @@
+# Experiments
+
+Fenix uses Mozilla's [Experiments service for experimentation](https://github.com/mozilla-mobile/android-components/blob/master/components/service/experiments/README.md).
+
+## Experiment Naming
+
+Naming must be unique and under 100 characters.
+
+Name experiments in the format of "fenix_experiment_issue", such as "fenix_etp_5651"
+
+Branches should be descriptive of the control and treatment.
+
+## Applying Experiment Branches
+
+The Fenix engineer should [check for if a user is in and experiment and then apply the branches of this experiment](https://github.com/mozilla-mobile/android-components/blob/master/components/service/experiments/README.md#checking-if-a-user-is-part-of-an-experiment).
+See [no-op experiment example](https://github.com/mozilla-mobile/fenix/pull/4551) and [ETP example](https://github.com/mozilla-mobile/fenix/pull/5723).
+
+## Experimenter
+
+[Experimenter](https://experimenter.services.mozilla.com/) is the web application for managing user experiments for Mozilla.
+Once the Fenix engineer has added the experiment branches to Fenix, they should update the experiment in Experimenter with the unique experiment name, unique branch names, and version code for experiment to take place.
+If the experiment requires new telemetry/metrics, a data review will have to be completed in Experimenter.
+
```

Comment:
I think it would be good to include a note about the "ready to ship" button, and that after it's been hit another team will then take the information from Experimentor and use it to set up the actual experiment.